manual_flagging/initial_flagging.py

```diff
@@ -121,153 +140,373 @@
 df = df.astype(conversion_dict)
 df["ptax_flag_original"].fillna(False, inplace=True)
 
-# Separate res and condo sales based on the indicator column
-df_res = df[df["indicator"] == "res"].reset_index(drop=True)
-df_condo = df[df["indicator"] == "condo"].reset_index(drop=True)
+# Filter to correct tris
+tri_stat_groups = {
+    tri: method
+    for tri, method in inputs["tri_stat_groups"].items()
+    if tri in inputs["run_tri"]
+}
 
-# Create condo stat groups. Condos are all collapsed into a single class,
-# since there are very few 297s or 399s
-condo_stat_groups = inputs["stat_groups"].copy()
-condo_stat_groups.remove("class")
+# Handle current methodology data manipulation if needed (temporary, need to find a better way)
+if "current" in tri_stat_groups.values():
+    # Calculate the building's age
+    current_year = datetime.datetime.now().year
+    df["bldg_age"] = current_year - df["yrblt"]
+
+    # Ingest new geographic groups
+    df_new_groups = pd.read_excel(
+        os.path.join(root, "QC_salesval_nbhds_round2.xlsx"),
+        usecols=["Town Nbhd", "Town Grp 1"],
+    ).rename(columns={"Town Nbhd": "nbhd", "Town Grp 1": "geography_split"})
+    df["nbhd"] = df["nbhd"].astype(int)
+    df = pd.merge(df, df_new_groups, on="nbhd", how="left")
+
+# - - - - - - -
+# Make correct filters and set up dictionary structure
+# - - - - - - -
+
+dfs_to_feature_creation = {}  # Dictionary to store DataFrames
+
+for tri, method in tri_stat_groups.items():
```

Comment:
This for loop looks at the `inputs_intitial.yaml` and creates a dictionary structure that is used and added on in the rest of the script.